src/gluonts/mx/distribution/genpareto.py

```diff
@@ -91,6 +91,24 @@ def genpareto_log_prob(x, xi, beta):
             genpareto_log_prob(F.abs(x), xi, beta),
```

Comment:
Could you fix also the `-np.inf` to `-1e-15`? Same for `Gamma`. This can be a separate pr too.

Reply:
Thanks Konstantinos, i amended the pr to include these.